### ISSUE TITLE
[v6r12] pilotCommands __doOSG does not work: AttributeError: 'PilotParams' object has no attribute 'installScript'

### ISSUE THREAD

**Issue by andresailer** [DIRACGrid/DIRAC#2462]
I have pilots running at IFCA-LCG2 now, which don't do anything, but end with this message:

```
2015-06-30 11:21:05 UTC INFO [Pilot] Command ConfigureSite instantiated from pilotCommands
/usr/bin/python
2.6.6 (r266:84292, Jan 22 2014, 05:06:49) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-3)]
('Linux', 'cloudprv-05-0.ifca.es', '2.6.32-504.3.3.el6.x86_64', '#1 SMP Wed Dec 17 18:21:33 CET 2014', 'x86_64')
'PilotParams' object has no attribute 'installScript'
```

The pilots run the __doOSG function, because have the environment variable
```
OSG_WN_TMP=/tmp/6225259.1.cloud.q
```
defined.
This line is then causing the error
https://github.com/DIRACGrid/DIRAC/blob/rel-v6r13/WorkloadManagementSystem/PilotAgent/pilotCommands.py#L579
```python
shutil.copy( self.pp.installScript, os.path.join( osgDir, self.pp.installScriptName ) )
```
self.pp.installScriptName also does not exist either by the way.

The pilots I have running on an actual OSG site (PNNL) don't have this problem, their environment in different.
Is this part of the code still necessary? It looks like it is a re-factoring remainder from the installation part of days gone by.

**Comment by fstagni** [comment]
You are probably correct, indeed this part is a copy paste of some very old code, but I would like to know what @atsareg says about it.

**Comment by andresailer** [comment]
Solved by #2503
